papermill/preprocess.py

```diff
@@ -49,10 +33,9 @@ class PapermillExecutePreprocessor(ExecutePreprocessor):
     # Copyright (c) IPython Development Team.
     # Distributed under the terms of the Modified BSD License.
 
-    # TODO: Delete this wrapper when nbconvert allows for setting preprocessor
-    # hood in a more convienent manner
+    # TODO: Delete this wrapper and push shared changes to nbconvert
```

Comment:
In order to make TODO bot do the issue tracking work for you in a nice clean way, switch from use of the word `this` and use something descriptive. "Delete the preprocess wrapper from PapermillExecutePreprocessor and push shared changes to nbconvert".